modules/apps/portal-vulcan/portal-vulcan-impl/src/test/java/com/liferay/portal/vulcan/internal/openapi/contributor/ExtensionsOpenAPIContributorTest.java

```diff
@@ -0,0 +1,387 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.internal.openapi.contributor;
+
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.odata.entity.EntityField;
+import com.liferay.portal.odata.entity.EntityModel;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+import com.liferay.portal.vulcan.openapi.OpenAPIContext;
+import com.liferay.portal.vulcan.resource.EntityModelResource;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.media.Schema;
+
+import java.lang.reflect.Field;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.Application;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+
+/**
+ * @author Magdalena Jedraszak
+ */
+public class ExtensionsOpenAPIContributorTest {
```

Comment:
I think we need an integration test. The point of this story is to ensure the OpenAPI contains the x-filterable attribute for the REST APIs. It would be interesting to verify in ObjectEntry REST API and in a general API, for example, Blog postings

Reply:
@ccorreagg Yeah, I was doubtful about it,  I was thinking about maybe including tests for Objects and the rest-builder layer in the *.ftl's template. But also if you say only one example is okay I'm fine with it :D

Reply:
Well, we could include in the REST Builder, but we need to include an EntityModel in the rest builder tests, please check if it is possible.

For Objects, I think we need it for sure, it is a good idea to include maybe in the Object Entry Resource Test class

Reply:
> For Objects, I think we need it for sure, it is a good idea to include maybe in the Object Entry Resource Test class
Yeah we almost don't have tests there :trollface: (vide https://liferay.slack.com/archives/C19PWQ74J/p1726560969541789?thread_ts=1724832421.187629&cid=C19PWQ74J)